options.go

```diff
@@ -173,6 +178,19 @@ func Statter(s log.StatsReporter) Option {
 	}
 }
 
+// Identity is used to specify an identity as this Ringpop instance's identity.
+// Since a ringpop instance is by default identified by its hostport, it's not allowed to manually specify a
+// hostport as its identity.
+func Identity(identity string) Option {
+	return func(r *Ringpop) error {
+		if util.HostportPattern.MatchString(identity) {
+			return errors.New("a hostport is not valid as an identity")
```

Comment:
Was thinking the same but all other errors in this file are created inline. I chose to follow the code style (of this file).